docs/source/gbd2019_models/risk_attributable_causes/protein_energy_malnutrition/index.rst

```diff
@@ -11,13 +11,150 @@ PEM is responsible for both fatal and nonfatal outcomes within the GBD
+
+Note that as PEM is defined as "a lack of dietary protein and/or energy", it 
+includes famines and severe droughts. These result in discontinuities in PEM 
+estimation, which the GBD team accounts for. The appendix specifically mentions 
+using the Tombstone report to estimate deaths due to the famine during the Great 
+Leap Forward in the 1960s in China. [GBD-2019-Capstone-Appendix-PEM]_
+
+GBD's nonfatal PEM model takes as its case definition "moderate and severe acute 
+malnutrition", defined in terms of distance from the mean WHZ score given by the 
+WHO 2006 growth standard for children. The relevant ICD 10 codes are E40-E46.9, 
+E64.0, and ICD 9 codes are 260-263.9. PEM is partitioned into the following four 
+sequelae: [GBD-2019-Capstone-Appendix-PEM]_
+
+.. list-table:: Nonfatal PEM Sequelae
+  :widths: 10 15 15 15 
+  :header-rows: 1
+  
+  * - Sequela Name
+    - WHZ range
+    - Clinical description
+    - Disability weights
+  * - Moderate wasting without oedema
+    - {WHZ_i | -3SD < WHZ_i < -2SD}
+    - Asymptomatic
+    - NA
+  * - Moderate wasting with oedema
+    - {WHZ_i | -3SD < WHZ_i < -2SD}
+    - Is very tired and irritable and has diarrhoea
+    - 0.051 (0.031–0.079)
+  * - Severe wasting without oedema
+    - {WHZ_i | WHZ_i < -3SD}
+    - Is extremely skinny and has no energy.
+    - 0.128 (0.082–0.183)
+  * - Severe wasting with oedema
+    - {WHZ_i | WHZ_i < -3SD}
+    - Is very tired and irritable and has diarrhoea. Is extremely skinny and has no energy.
+    - 0.051 (0.031–0.079); 0.128 (0.082–0.183)
+
+These are mapped onto clinically-defined wasting states as follows:
+
+.. list-table:: Clinical definitions
+  :widths: 5 10
+  :header-rows: 1
+  
+  * - Condition
+    - Estimated by GBD sequelae
+  * - Kwashiorkor
+    - {Moderate wasting with oedema} + {Severe wasting with oedema}
+  * - Marasmus
+    - {Severe wasting without oedema} + {Severe wasting with oedema}
+
+The nonfatal estimation pipeline comprises five models:
+
+.. list-table:: Nonfatal PEM sub-models
+  :widths: 15 5 5
+  :header-rows: 1
+  
+  * - Modeled entity
+    - Age
+    - Modeling software
+  * - Prevalence of WHZ <-2SD
+    - under-5
+    - STGPR
+  * - Prevalence of WHZ <-3SD
+    - under-5
+    - STGPR
+  * - Proportion of WHZ <-2SD with oedema
+    - under-5
+    - DisMod
+  * - Proportion of WHZ <-3SD with oedema
+    - under-5
+    - DisMod
+  * - All WHZ <-2SD (PEM)
+    - All ages
+    - DisMod
+
+For the all-age model, they set the duration of PEM to 9 months after consulting 
+with nutrition experts. The current modelers (as of June 2021 no longer have 
+documentation of these conversations, which took place sometime before 2015). 
+They used a remission rate of 0.25 - 1.25 (remitted cases of PEM per person-year 
+of illness). Note this is a rather wide interval that allowed DisMod to choose a 
+remission rate within the given bounds based on other input data. [GBD-2019-Capstone-Appendix-PEM]_
+
```

Comment:
Perhaps we can make a note here to say this remission bound is so wide that the modellers are letting dismod come up with a remission rather than bounding it within some range. Additionally, we can make a note saying that our model might not match 9 months of PEM duration and therefore the incidence of PEM from our wasting model may not match that of PEM. Once we solve our equations, we can cross compare and see.

Reply:
@Ninicorn - hm. i think we had different understandings of what the modelers told us? i think Ryan said that they specifically feed a wide interval (0.25 - 1.25) into DisMod _so_ that DisMod will not consider that a constraint when choosing other values (incidence, remission, etc), and allowing DisMod to choose some remission rate that works.

^ As opposed to, they found in the literature a super wide range, and so didn't enter anything into DisMod and let it choose freely:

From the appendix:
> Using available information from scientific publications, which suggest the mean duration of illness is
nine months, and conversations with collaborators and nutrition experts, we applied what we consider a
plausible set of remission rate bounds of 0.25–1.25 (# of remitted cases of PEM per person-year of
illness) to the final of the five models. These bounds allowed DisMod to mathematically derive an
internally consistent solution for incidence, prevalence, remission, excess mortality, and cause-specific
mortality using all available data

Reply:
Perhaps I wasn't clear in my description, but my understanding matches yours and I was trying to convey the same meaning as you...I see how it was not very clear now. sorry :(